queries/cpp/injections.scm

```diff
@@ -1,3 +1,19 @@
 (preproc_arg) @cpp
 
 (comment) @comment
+
+(call_expression
```

Comment:
It's not necessary to repeat the queries for cop as they get inherited from c

Reply:
I noticed the other queries were repeated so I did the same.

Reply:
Oh, you're completely right. The injections are repeated because nvim does not yet support self injections. So the macro self injection needs to be repeated. Also cuda and hlsl inherit from c++ and hlsl from C. Glsl has shader printf: `debugPrintfEXT`